.github/workflows/node.js.yml

```
@@ -3,8 +3,10 @@
 name: Node.js CI
 
 on:
-  - push
-  - pull_request
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
```

For compat with jQuery, maybe it should be:
```suggestion
  pull_request:
  push:
    branches-ignore: "dependabot/**"
```
Thoughts?

---

Updated.